news/_posts/releases/2020-10-22-prestashop-1-7-7-0-rc-release.md

```diff
@@ -0,0 +1,99 @@
+This brings us to more than 1300 merged pull requests for the 1.7.7.0 milestone, it is definitely our biggest yet.
+
+If you are looking for more details about all changes and new features expected in 1.7.7, the [1.7.7 beta version release note](https://build.prestashop.com/news/prestashop-1-7-7-0-beta-release/) is the perfect article for you to read!
+
+
+## Download
+
+You can download PrestaShop 1.7.7 RC1 here:
+
+[Download PrestaShop 1.7.7 RC1 now!](https://www.prestashop.com/en/developers-versions){: .btn .btn-lg .btn-success}
+{: .text-center}
+
+
+## How to help finish the 1.7.7
+
+Your feedback on this RC is essential, the regressions you find right now are as many bugs less on the final version and less patch versions. It also means fewer problems on your (or your customer’s) online store. 
+
+You must:
+
+- Install it and test that there is no regression compared to the 1.7.6 version
+- If you develop modules or themes, test them on this version
+- Test the auto-upgrade plugin
+- [Report regressions on GitHub](https://github.com/PrestaShop/PrestaShop/issues) (read how to [report issues](https://devdocs.prestashop.com/1.7/contribute/contribute-reporting-issues/))
+- Fix regressions by creating a [pull request](https://github.com/PrestaShop/PrestaShop/compare) (read the [contribution guidelines](https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/))
+
+
+If everything is working well, feel free to comment on this article to tell us, and share on social networks, we also need this kind of feedback! ;-)
+
+## Acknowledgments
+
+A huge thanks to everyone who got involved by reporting, testing and fixing regressions during the first beta phase.
+Internal and external contributors who submitted pull requests and participated to build this release candidate: [PululuK](https://github.com/PululuK), [okom3pom](https://github.com/okom3pom), [ks129](https://github.com/ks129), [Boubker Bribri](https://github.com/boubkerbribri), [Franck Lefèvre](https://github.com/Progi1984), [Ibrahima Sow](https://github.com/sowbiba), [Jonathan Lelievre](https://github.com/jolelievre), [Mathieu Ferment](https://github.com/matks), [Matthieu Rolland](https://github.com/matthieu-rolland), [Nesrine Abdmouleh](https://github.com/nesrineabdmouleh), [Pablo Borowicz](https://github.com/eternoendless), [Pierre Rambaud](https://github.com/PierreRambaud), [Simon Garny](https://github.com/SimonGrn), [Stephane Decisy](https://github.com/SD1982), [Thomas Baccelli](https://github.com/atomiix) and [Valentin Szczupak](https://github.com/NeOMakinG). Thank you!
+
+External contributors who reported regressions to build a more stable 1.7.7: [ks129](https://github.com/ks129), [okom3pom](https://github.com/okom3pom), [MathiasReker](https://github.com/MathiasReker) and [kpodemski](https://github.com/kpodemski). Thank you!
```

Comment:
Well, @kpodemski is no more an external contributor 😛 he's now a maintainer

Reply:
how do we handle that?
I mean, he is still not a PrestaShop employee or a subcontractor... And, historically, the "by / thank you" was made to make a difference.
Then, should this change, and starting for now, we use that to make a difference between maintainers and contributors instead ?
what about PrestaShop employees who are not maintainers, like QA and PM?